### PR TITLE
Add automation doc in markdown form

### PR DESCRIPTION
**Pull request by RobGallo**
#### Describe the change

Adding documentation for using Axe.Windows automation.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: docs/Automation.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+CommandConstStrings.NoViolationPolicy | “NoViolationPolicy” | Specifies how output files should be handled if no violations are found. <ul><li>**Discard** will discard the results file.<li>**Retain** will retain the results file.</ul>*Default value = **Retain***. Valid only as a Tier 3 setting.
+CommandConstStrings.Discard  | “Discard” | Value for NoViolationPolicy.
+CommandConstStrings.Retain | “Retain” | Value for NoViolationPolicy.
+CommandConstStrings.TeamName | “TeamName” | Value for Team name in telemetry.
+
+### Using the assembly
+You can get the files via a NuGet package Configure NuGet to retrieve the
+**Axe.Windows** package from
+<https://TBD/nuget/v3/index.json>,
```
Comment (DaveTryon) [inline docs/Automation.md]: Do we have the correct (version-agnostic) path here?

File: docs/Automation.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+
+#### From PowerShell
+-   Start your PowerShell script from the folder where the AxeWindows
+    DLLs are located.
+-   Load the assembly via **Import-Module Axe.Windows.Automation.dll**.
+-   Access the Start command via the **Start-AxeWindows** cmdlet.
+-   Access the Snapshot command via the **Invoke-Snapshot** cmdlet.
+-   Access the Stop command via the **Stop-AxeWindows** cmdlet.
+
+Sample PowerShell script (the 2 second delay exists to give the app time to initialize before it is scanned):
```
Comment (DaveTryon) [inline docs/Automation.md]: This script will work only if the instance of notepad.exe that we start is the only instance in the entire system. That's a reasonable limitation for a sample script, although we probably ought to document it as such

File: docs/Automation.md (hunk 1 above)
Comment (RobGallo) [reply docs/Automation.md]: Updated.

File: docs/Automation.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
+## Axe.Windows - Automation
+
+### Overview
+To enable various automation scenarios, we have created an assembly
+(`Axe.Windows.Automation.dll`) that exposes a subset of core
+AxeWindows functionality to automation systems. This can be used in
+a variety of ways, including as part of a standalone test system, as part of a
```
Comment (waabid) [inline docs/Automation.md]: nit: 'as part of a' is used redundantly here and can be replaced this with:

'including as part of a standalone test system, PowerShell script, or CI/CD solution'

File: docs/Automation.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+PowerShell script, or as part of a CI/CD solution. One or more scans can be
+performed during a test run, and the outputs will be saved to disk. No UI is
```
Comment (waabid) [inline docs/Automation.md]: nit: 'during a test run, and the outputs...'

The comma here is unnecessary.

Comment (RobGallo) [reply docs/Automation.md]: I brake for grammarians!

**Pull request merged**